lists/TN-RNE

```diff
@@ -0,0 +1,52 @@
+{
+  "name": {
+    "en": "The National Register of Companies",
```

Comment:
Maybe edit this to "The National Register of Companies, Tunisia"

Or "Tunisian National Register of Companies"